### ISSUE TITLE
Dialogs visual representation

### ISSUE THREAD

**Issue by marcpascualsanchez** [axa-group/nlp.js#833]
Is there any way to have a visual representation of the dialogs?
I have worked with the app from https://github.com/axa-group/nlp.js-app and it is great, but I can't find an option to configure dialogs.
If there is not currently a way to do this, are there any guidelines to contribute with the project? This would be a great addition.

The more I learn about this library, the more complete and useful it feels. Great work.

**Comment by jesus-seijas-sp** [comment]
Short answer... not opensourced. But.... you have several options:
- You have https://nodered.org/ There you can look for different node-red components that integrate NLP.js: https://www.npmjs.com/package/node-red-contrib-nlp https://www.npmjs.com/package/node-red-contrib-viseo-nlp-js
- Blockly: it's easy to integrate to transpile it's code to a given language. https://developers.google.com/blockly
- Microsoft Composer: https://github.com/microsoft/BotFramework-Composer I did an integration (non opensource) with NLP.js and it works.

I have experience with the blockly and Microsoft Composer ones, I can contribute if someone creates an open source around this.

**Comment by marcpascualsanchez** [comment]
In my case what I was looking for is something like Microsoft Composer, not Blockly nor node-red. After testing Microsoft Composer, which did not feel newbie frienldy nor easy to integrate, I tried other options.

After many tries, I have created an integration with [Botpress](https://botpress.com/). It is in my opinion more friendly and easier to create and mantain chatbot flows, and the bot training files are clear and easy to understand. So I ended up parsing these files into NLP.js files.
